content/updates/2020-09/2-Quick-Export-revamped-UI.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/Quick-export-new-UI_Export-with-labels.png
+pim_announcement_alt_img: "Quick Export revamped UI"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Quick Export revamped UI
+::: meta-data type="New" features="Collaboration, Productivity" available="mid September" link-to-doc="../articles/quick-export.html"
+:::
+
+Discover a fresh UI for the Quick Export!
```

Comment:
Do you need to go to the next line at the end of this sentence? If not, please put the next sentence on the same line, otherwise, add two spaces at the end of your line.
And remember that the first paragraph will be displayed in the PIM. ;)